sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -144,16 +144,16 @@ namespace Azure { namespace Core { namespace Http {
      * @param httpMethod HTTP method.
      * @param url URL.
      * @param bodyStream #Azure::Core::IO::BodyStream.
-     * @param downloadViaStream A boolean value indicating whether download should happen via
-     * stream.
+     * @param bufferedDownload A boolean value indicating whether download should use a buffer
+     * for the response or return a body stream instead.
      */
     explicit Request(
         HttpMethod httpMethod,
         Url url,
         Azure::Core::IO::BodyStream* bodyStream,
-        bool downloadViaStream)
+        bool bufferedDownload)
```

Comment:
nit: I'd name the parameter (here and below) `isBuffferedDownload`

Reply:
Name suggested by Rick before. Already changed a couple of times.  ahahha..  Let's keep this one for now

Reply:
Can I recommend another name?

`dontUseThisBoolItsOnlyForSomethingSomethingBuffered`